src/modal-config.ts

```diff
@@ -0,0 +1,76 @@
+import { TemplateResult, html } from 'lit-html';
+
+/**
+ * Configuration to show a modal
+ *
+ * @export
+ * @class ModalConfig
+ */
+export class ModalConfig {
+  /**
+   * The title that shows in the header
+   *
+   * @type {(TemplateResult | undefined)}
```

Comment:
Wouldn't this be `TemplateResult | LitHtml object`?

Reply:
`TemplateResult` is the type that gets output when you use ``` html`Foo <b>Bar</b>` ``` so in Typescript, the `render()` method looks like this:
```
render(): TemplateResult {
  return html`<div> ... </div>`;
}
```

Technically, you can also pass in strings here, but the `TemplateResult` lets you put markup in the field.

Reply:
Ah, ok. So that's the former object type. But if you're setting it to a default TemplateResult value, shouldn't the doc line be `@type {TemplateResult}`?

Reply:
Yeah I set a default value, but this lets people set it to `undefined` if they don't want the default.

Reply:
I removed the default value.